doc/md/templates.md

```diff
@@ -71,6 +71,50 @@ In order to override an existing template, use its name. For example:
 {{ end }}
 ```
 
+## Helper Templates
+
+As mentioned above, `ent` writes the template execution output to a file named as the template.
+For example, a template defined as `{{ define "stringer" }}` will be written to a file named `ent/stringer.go`.
```

Comment:
```suggestion
As mentioned above, `ent` writes each template's execution output to a file named the same as the template.
For example, the output from a template defined as `{{ define "stringer" }}` will be written to a file named `ent/stringer.go`.
```

Reply:
I think it has to be `...named the same like the template.`. I could be wrong though, not my native language.